content/blog/2016/2016-05-26-introducing-blue-ocean.adoc

```diff
@@ -0,0 +1,140 @@
+and start automating any  Branches and validating Pull Requests.
+
+Jenkins will report the status of your pipeline right inside Github or
+Bitbucket on all your commits, branches or pull requests.
+
+image:/images/post-images/blueocean/pr-view.png[Pull request view, role=center]
+
+
+==== Personalised View
+
+Favourite any pipelines, branches or pull requests and see them appear on your
+personalised dashboard. Intelligence is being built into the dashboard. Jobs
+that need your attention, say a Pipeline waiting for approval or a failing job
+that you have recently changed, appear on the top of the dashboard.
+
+
+image:/images/post-images/blueocean/personalized-dashboard.png[Personalized dashboard, role=center]
+
+
+You can read more about Blue Ocean and its goals on the
+link:/projects/blueocean[project page] and developers should watch the
+link:/content/mailing-lists[Developers list] for more information.
+
+---
+
+For Jenkins developers and plugin authors:
+
+==== Jenkins Design “Language”
+
+The Jenkins Design Language (JDL) is a set of standardised React components and
+a style guide that help developers create plugins that retain the look and feel
+of Blue Ocean in an effortless way. We will be publishing more on the JDL,
+including the style guide and developer documentation, over the next few weeks.
+
+==== Modern JavaScript toolchain
+
+The Jenkins plugin tool chain has been extended so that developers can use
+link:https://medium.com/@rajaraodv/5-javascript-bad-parts-that-are-fixed-in-es6-c7c45d44fd81[ES6],
+link:https://facebook.github.io/react/[React], link:https://www.npmjs.com/[NPM]
+in their plugins without endless yak-shaving. Jenkins
+link:https://github.com/jenkinsci/js-modules[js-modules] are already in use in
+Jenkins today, and this builds on this, using the same tooling.
+
+==== Client side Extension points
+
+Client Side plugins use Jenkins plugin infrastructure. The Blue Ocean libraries
+built on ES6 and React.js provide an extensible client side component model
+that looks familiar to developers who have built Jenkins plugins before. Client
+side extension points can help isolate failure, so one bad plugin doesn’t take
+a whole page down.
+
+==== Server Sent Events
+
+link:https://developer.mozilla.org/en-US/docs/Web/API/Server-sent_events/Using_server-sent_events[Server Sent Events]
+(SSE) allow plugin developers to tap into changes of state on the server and make
+their UI update in real time (link:https://www.youtube.com/watch?v=EttzK5OOpv0[watch this for a
+demo]).
+
+The Blue Ocean project will only succeed with enough interest and support from
+Jenkins developers and plugin authors, your help is needed, please do join in
+on the link:http://groups.google.com/forum/#!forum/jenkinsci-dev[Jenkins Developer mailing list].
+
+
+== Links
+
+* link:/projects/blueocean[Blue Ocean project page]
+* link:https://github.com/cloudbees/blueocean[Blue Ocean GitHub repository]
```

Comment:
The HOSTING ticket has not yet been filed by @michaelneale 

This will be updated when that goes through